### DISCUSSION TITLE
Scrollycoding Mobile responsiveness

### DISCUSSION THREAD

**Discussion by junaid33** [code-hike/codehike#218] [Q&A] [answered]
Hello, I'm using codehike and nextra for our documentation and it is working wonderfully. 

I had found out about codehike a year ago and when I checked the examples then, it was mobile responsive. In the Scrollycoding example, the code block would show up on the right side on desktop and on the bottom for mobile. This doesn't seem to be the case anymore. Can this be enabled? Maybe nextra is throwing off the responsiveness?

**Comment by pomber** [comment, marked as answer]
Maybe you are thinking of [this blog post](https://pomb.us/build-your-own-react/) which isn't actually using Code Hike. 

I'm currently working on improving the Scrollycoding mobile experience, but it will be more like [this example](https://code-hike-scrollycoding-preview.vercel.app/posts/lorem-ipsum-three).

Update: This was released in version `v0.7.0`. [Demo here](https://twitter.com/pomber/status/1547247631776321537).

**Reply by junaid33** [reply]
Wow, looks great! Thanks
